regex-assembly/932125.ra

```diff
@@ -144,39 +156,49 @@
     ipcsv@
     ipmo@
     ipsn@
+    irm@
+    ise@
     iwmi@
     iwr@
     ##! disabled for FP: kill
-    lp@
+    ls
+    man@
+    md@
     ##! disabled for FP: measure
     mi@
+    mount@
     ##! disabled for FP: move
     mp@
+    mv@
     nal@
     ndr@
     ni@
     nmo@
+    npssc
     nsn@
     nv@
     ogv@
     ##! disabled for FP: oh
     popd@
     pushd@
     ##! disabled for FP: pwd
+    ##! disabled for FP: r
     rbp@
     rcjb@
+    rcsn
     rd@
     rdr@
     ren@
     ri@
     rjb@
+    rm@
     rmdir@
     rmo@
     rni@
     rnp@
     rp@
     rsn@
-    rsnp@
```

Comment:
Is this from an older version of Powershell? Maybe we should keep it...

Reply:
Well... re-added.